x-pack/plugins/cases/server/common/utils.ts

```diff
@@ -142,14 +142,14 @@ export const transformComments = (
 
 export const flattenCommentSavedObjects = (
   savedObjects: Array<SavedObject<CommentAttributes>>
```

Comment:
We can create the type `type CommentSavedObjectTransformed = SavedObject<AttachmentTransformedAttributes>` and use it here.

Reply:
These utilities are used by the client, I'll create the type you recommended and use it within the service layer 👍